packages/eth2.0-spec-test-util/src/bulk.ts

```diff
@@ -0,0 +1,107 @@
+import {load} from "js-yaml";
+import {readFileSync, writeFile} from "fs";
+import {schema} from "./yaml/schema";
+import {describe} from "mocha";
+import {objectToCamelCase} from "./util";
+import {expect} from "chai";
+import profiler from "v8-profiler-next";
+
+export interface BaseCase {
+  description: string;
+}
+
+interface TestSpec<TestCase extends BaseCase> {
+  title: string;
+  summary: string;
+  forksTimeline: string;
+  forks: string;
+  config: string;
+  runner: string;
+  handler: string;
+  testCases: TestCase[];
+}
+
+const env = process.env;
+
+/**
+ * Run yaml Eth2.0 bulk spec tests for a certain function
```

Comment:
Bulk vs. ?
I guess this is still built for the old all-in-one-yaml style.

Reply:
@wemeetagain name suggestion. I'm planning to switch to `describeFileSpec` and `describeDirectorySpec`. Can't think of better naming

Reply:
maybe `describeSingleSpec` / `describeMultiSpec` ?

Reply:
which one is which? :laughing: I mean only diff is in old one, test cases are in single file and in new one test cases are in directory

Reply:
this would be `describeMultiSpec`